gomanifest/internal/manifest.go

```diff
@@ -49,19 +49,35 @@ func transformVersion(inVersion string) string {
 
 // getPackageName ... Utility function to convert package + module data into package name used by manifest.
 func getPackageName(depPackage DepPackage) string {
-	// Get module / package@module
+	var modulePath string = depPackage.Module.Path
```

Comment:
```suggestion
	modulePath  := depPackage.Module.Path
```
Isnt this the preferred way of variable assignment?

Reply:
@prashbnair , := will declare and assign variable.

Reply:
Both are idiomatic go,

```
var a int64
if .. {
   a = 1
} else {
  a = 2
}
```

```
a := 1
if .. {
   a = 2
}
```

Reply:
This is done.